packages/playwright-core/src/utils/tar.ts

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import fs from 'fs';
+import { Writable, once } from 'stream';
+import assert from 'assert';
+
+enum TarType {
+  REGTYPE,
```

Comment:
Nope Copilot - they are correctly assigned! REGTYPE is set to 0, SYMTYPE to 2, and so forth. And because we parseInt the type, it ends up as numbers and not as strings with numbers inside.